labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,7 +33,13 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+
+    val input = "ejp mysljylc kd kxveddknmc re jsicpdrysi rbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcd de kr kd eoya kw aej tysr re ujdr lkgc jv"
+    val output = "our language is impossible to understand there are twenty six factorial possibilities so it is okay if you want to just give up"
+
+    val tuples = input.zip(output)
+    val m = tuples.toMap
+    lines.toList.map(_.map(m(_)))
```

Comment:
ah, I missed that. I would just do `val tuples = 'u'->'z'::'z'->'u'::input.zip(output)` though